### PR TITLE
Minor scheduler clean-up

### PR DESCRIPTION
**Pull request by kgaillot**
With the recent colocation code changes, a couple areas of code can be cleaned up better

### CODE REVIEW

**Comment by kgaillot** [comment]
@nrwahl2 , can you review this follow-up to the colocation changes? Low priority

File: lib/pacemaker/pcmk_sched_clone.c [hunk 1]
```diff
@@ -28,12 +28,15 @@ add_dependent_scores(gpointer data, gpointer user_data)
...
 
     if (pcmk__colocation_has_influence(colocation, NULL)) {
-        pcmk__add_colocated_node_scores(colocation->dependent, clone->id,
-                                        &clone->allowed_nodes,
-                                        colocation->node_attribute,
-                                        colocation->score / (float) INFINITY,
-                                        pcmk__coloc_select_active
-                                        |pcmk__coloc_select_nonnegative);
+        pe_resource_t *rsc = colocation->dependent;
+        const float score = colocation->score / (float) INFINITY;
```
Comment (nrwahl2) [inline lib/pacemaker/pcmk_sched_clone.c:32]: Should we call this `factor`? Doesn't really matter

File: lib/pacemaker/pcmk_sched_instances.c [hunk 2]
```diff
@@ -183,27 +183,33 @@ apply_parent_colocations(const pe_resource_t *rsc, GHashTable **nodes)
 {
     GList *iter = NULL;
     pcmk__colocation_t *colocation = NULL;
+    pe_resource_t *other = NULL;
+    float score = 0.0;
```
Comment (nrwahl2) [inline lib/pacemaker/pcmk_sched_instances.c:187]: ditto

**Review by nrwahl2 — APPROVED** [review]
Looks good to me

File: lib/pacemaker/pcmk_sched_clone.c (hunk 1 above)
Comment (kgaillot) [reply lib/pacemaker/pcmk_sched_clone.c:32]: Whoops, yes

**Review by nrwahl2 — APPROVED** [review]

**Pull request merged**